### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.11 es 4.79 (18 novembre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#330] [Esercizi M-Z]
![es79](https://github.com/user-attachments/assets/0825d5fc-f202-4d60-83bd-a95b22a56469)

**Comment by alem1105** [comment]
<img width="1030" alt="image" src="https://github.com/user-attachments/assets/6e194e59-7c9f-4b1d-af4f-fa4f65fc41e5">

**Reply by alem1105** [reply]
**Mancava -P(X=2) 😔**

<img width="1110" alt="image" src="https://github.com/user-attachments/assets/6cce5860-a786-40fa-b7a1-b7ab168efa22">
